### ISSUE TITLE
Updating angular app from version 12 to version 13.

### ISSUE THREAD

**Issue by ffadikhan** [angular/angular-cli#22143]
### Which @angular/* package(s) are the source of the bug?

compiler-cli, upgrade

### Is this a regression?

No

### Description

Hi team #angular-cli #nodejs, I am updating my large application from angular 12 to angular 13.
Update node version from 14.18.0 to 16.13.0 through nvm. 
When I excecute command.
**ng update @angular/cli @angular/core --force**
it give mentioned below error.

 **```
Package successfully installed.
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'C:\Users\Fahad\AppData\Local\Temp\angular-cli-packages-PAYCCt\node_modules\@angular\cli\bin\ng.js''
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```**
I updated cache  using mentioned below commands but nothing is progress.

**npm cache verify
npm cache clean --force**


### Please provide a link to a minimal reproduction of the bug

_No response_

### Please provide the exception or error you saw

_No response_

### Please provide the environment you discovered this bug in

```true
Angular CLI: 13.0.1
Node: 16.13.0
Package Manager: npm 8.1.0
OS: win32 x64

Angular:
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1300.1 (cli-only)
@angular-devkit/core         13.0.1 (cli-only)
@angular-devkit/schematics   13.0.1 (cli-only)
@schematics/angular          13.0.1 (cli-only)
```

**Comment by alan-agius4** [comment]
@ffadikhan did you run `npm install` in your workspace directory prior of running `ng update`?

**Comment by ffadikhan** [comment]
> @ffadikhan did you run `npm install` in your workspace directory prior of running `ng update`?

I did it as u mentioned. But facing same issue.
Is it possible that this issue is also belongs to our previous updating?

**Comment by alan-agius4** [comment]
Can you try to run the update using the below command

```
cmd /C "set "NG_DISABLE_VERSION_CHECK=1" && ng update @angular/cli @angular/core --force
```

**Comment by ffadikhan** [comment]
> Can you try to run the update using the below command
> 
> ```
> cmd /C "set "NG_DISABLE_VERSION_CHECK=1" && ng update @angular/cli @angular/core --force
> ```

@alan-agius4  I try above command that mentioned you but error still there.

**Comment by alan-agius4** [comment]
In that case we’ll need a reproduction or you could also share your project privately so that we can investigate this issue further.

**Comment by ffadikhan** [comment]
> In that case we’ll need a reproduction or you could also share your project privately so that we can investigate this issue further.

@alan-agius4 thanks for your cooperation. Issue has been solved after angular version updated from 13.0.1 to 13.0.2. Excecuted below mentioned command and migrate into v13 successfully.

**npm uninstall -g @angular/cli @angular/core

npm cache verify

npm cache clean --force

npm install -g @angular/cli@latest

npx @angular/cli@13 update @angular/core@13 @angular/cli@13 --force**

And installed **node v14** instead of **v16**.

**Comment by alan-agius4** [comment]
Closing as issue is resolved as per above comment.
